core/src/main/java/org/sonarsource/sonarlint/core/analyzer/sensor/SonarLintSensorStorage.java

```diff
@@ -96,9 +98,11 @@ public void store(Issue issue) {
     String type = rule.type().name();
 
     List<org.sonarsource.sonarlint.core.client.api.common.analysis.Issue.Flow> flows = mapFlows(issue.flows());
+    List<QuickFix> quickFixes = issue instanceof DefaultSonarLintIssue ?
```

Comment:
The issue we get in this method should always be a `DefaultSonarLintIssue` so we could cast early